### PR TITLE
Adding in comments and analytics

### PR DESCRIPTION
**Pull request by mattjburrows**
Added the following:
- Commenting via Disqus
- Google analytics
- Canonical URL

### CODE REVIEW

File: _includes/comments.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div id="disqus_thread"></div>
+<script>
+// var disqus_developer = 1;
```
Comment (mattjburrows) [inline _includes/comments.html]: @nspragg would you prefer this to be dynamic?

e.g.

```
var developer_mode = (window.location.host === '127.0.0.1:4000');
var disqus_developer = developer_mode ? 1 : 0;
```

Comment (nspragg) [reply _includes/comments.html]: 👍

**Comment by nspragg** [comment]
@mattjburrows LGTM

**Comment by nspragg** [comment]
@mattjburrows thanks!

**Pull request merged**